src/components/material/material-buttons/physical/MaterialButtonsFindOnShelf.tsx

```diff
@@ -42,19 +42,6 @@ const MaterialButtonsFindOnShelf: FC<MaterialButtonsFindOnShelfProps> = ({
   }
 
   if (size !== "small") {
-    if (!data[0].available) {
-      return (
-        <Button
-          label={t("materialIsLoanedOutText")}
-          buttonType="none"
```

Comment:
Please consider if fetching availability data with `useGetAvailabilityV3()` on L24 in this file can be removed now.

As far as I can tell with this change the only place the data was actually used is gone. If that is the case then the request is obsolete.

Reply:
That is absolutely correct. It's removed now.